test/user.test.js

```diff
@@ -2,16 +2,17 @@ const
     request = require('supertest'),
     app = require('../server');
 
-describe("Create forum user dummy test", function() {
-    it("should return: { dummyTest: 'userCreate() dummy test passes' }", function(done) {
+describe("Create forum user test", function() {
+    it("should return: status 200", function(done) {
         request(app)
             .post('/api/v1/users')
-            .send({ dummyTestInput: 'this text is useless' })
-            .expect({ dummyTest: 'userCreate() dummy test passes' })
-            .end(function(err, res) {
-                if (err) done(err);
-                done();
-            });
+            .send({
+                username: 'Bob123',
+                name: 'bob',
+                email: 'bob420@hotmail.com',
+                hashedPassword: '324hkjhh92bfd8g1b@#$Fn912bf'
+            })
+            .expect(200, done);
```

Comment:
The issue here is the test will return 400 for every attempt after the first successful one because the database will not allow a duplicate user to be added.

Reply:
Ah yup, that is true. Thanks. I've checked with the DB and I can see that it's correctly posting to the DB, so I'll just leave it as is for now? and then I can update the unit test to delete it after the delete HTTP request has been created?

Reply:
I can confirm that the post is working as I tested it.